CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Improvements in verification of Audit consistency proofs
```

Comment:
v3.9.0 has already been released, don't add more release notes to it. This should go in a new section under "Unreleased".